### ISSUE TITLE
Akka HTTP, docs: show how to get collect some header (maybe cookies)

### ISSUE THREAD

**Issue by ktoso** [akka/akka-http#205]
<a href="https://github.com/ktoso"><img src="https://avatars.githubusercontent.com/u/120979?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ktoso](https://github.com/ktoso)**
_Wednesday Sep 30, 2015 at 08:22 GMT_
_Originally opened as https://github.com/akka/akka/issues/18604_

---

Via akka-user thread "Akka Http Client - Unmarshal Cookies".
For newcomers it's not clear how to "get all cookies" or other headers in a typesafe manner,
solution is to:

``` scala
Flow[HttpResponse].map(_.headers.collect { case `Set-Cookie`(x) ⇒ x})

```

We should have an example like this in client docs.

**Comment by nikom1337** [comment]
Hi, I would like to work on that issue, is anyone working on it already?

**Comment by ktoso** [comment]
I believe it's available, thanks

**Comment by jrudolph** [comment]
Fixed by #1719.
